projects/extension/sql/idempotent/013-vectorizer-api.sql

```diff
@@ -174,14 +196,25 @@ begin
     , grant_to
     );
 
-    -- create the target table
-    perform ai._vectorizer_create_target_table
-    ( _source_pk
-    , target_schema
-    , target_table
-    , _dimensions
-    , grant_to
-    );
+    if skip_chunking then
+        -- Add embedding column to source table
+        perform ai._vectorizer_add_embedding_column
+        ( _source_schema
+        , _source_table
+        , _dimensions
+        , embedding_column
+        , grant_to
+        );
```

Comment:
Should we dynamically decide an available column name here? Otherwise running create_vectorizer for same_table embeddings multiple times will error out if you don't provide a custom column_name.